src/Microsoft.Diagnostics.Monitoring.WebApi/Models/CallStackResults.cs

```diff
@@ -10,16 +10,16 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi.Models
     public class CallStackFrame
```

Comment:
NOTE:

The `MethodName`, `TypeName`, `ModuleName`, and `ThreadName` should be marked with the `[Required]` attributes. However that is beyond the scope of this PR effort and also wouldn't make any difference to our `openapi.json` since these types aren't included.